modules/bootstrap/variables.tf

```diff
@@ -15,3 +15,9 @@ variable "service_account" {
   default     = null
   type        = string
 }
+
+variable "location" {
+  description = "Location in which the GCS Bucket will be deployed. Available locations can be found under https://cloud.google.com/storage/docs/locations."
+  default     = "us"
```

Comment:
```suggestion
```

Reply:
Let's not have defaults here so the user has to explicitly specify the region they want to create the bucket in